.rubocop.yml

```diff
@@ -108,3 +111,10 @@ Performance/Caller:
 
 Metrics/BlockLength:
   Max: 193
+
+# TODO: Reconsider this in the future
+Performance/UnfreezeString:
+  Enabled: false
+
+Style/SafeNavigation:
+  Enabled: false
```

Comment:
Can we revert all these now and revisit?

Reply:
No, `rubocop lib` breaks the build.

Reply:
Surprised me as well. On `main` locally:
```
$ rubocop lib
74 files inspected, 85 offenses detected
```
[On CI](https://github.com/rspec/rspec-core/runs/1529378736):
```
bin/rubocop lib
74 files inspected, no offenses detected
```
The version is the same, 0.52.1.

Reply:
```
➜  rspec-core git:(main) ✗ bundle exec rake rubocop                                                                                                                                                                             (git)-[main]
bundle exec rubocop lib
Inspecting 74 files
..........................................................................

74 files inspected, no offenses detected
➜  rspec-core git:(main) ✗ bundle exec bin/rubocop lib                                                                                                                                                                          (git)-[main]
Inspecting 74 files
..........................................................................

74 files inspected, no offenses detected
➜  rspec-core git:(main) ✗ ruby -v                                                                                                                                                                                              (git)-[main]
ruby 2.7.1p83 (2020-03-31 revision a0c7c23c9c) [x86_64-darwin19]
```

Reply:
```
➜  rspec-core git:(4-0-dev) ✗ bundle exec bin/rubocop lib                                                                                                                                                                    (git)-[4-0-dev]
Inspecting 74 files
..........................................................................

74 files inspected, no offenses detected
➜  rspec-core git:(4-0-dev) ✗ ruby -v                                                                                                                                                                                        (git)-[4-0-dev]
ruby 2.7.1p83 (2020-03-31 revision a0c7c23c9c) [x86_64-darwin19]
```

Reply:
I reverted this fie and ran it on you branch, rubocop passed, can you provide a build which failed without this change? Otherwise please revert.